blockchain/transaction_svc.py

```diff
@@ -169,6 +176,8 @@ def run():
     hdlrs = [
         (r"^/transaction$", TransactionHandler),
         (r"^/transaction/(.*)", TransactionHandler),
+        (r"^/public_transmit$", public_transmission_handler),
```

Comment:
They were from the implementation of the RestAPI transmission call before we decided to make a separate ticket. Removed.